crates/rpc/src/lib.rs

```diff
@@ -68,14 +73,13 @@ Hint: If you are looking to run two instances of pathfinder, you must configure
             })?;
         let local_addr = server.local_addr()?;
 
-        let module_v02 = v02::register_methods(self.context.clone())?;
-        // FIXME add path handling
-        let _pathfinder_module = pathfinder::register_methods(self.context.clone())?;
-        let _module_v03 = v03::register_methods(self.context)?;
+        let module = crate::module::Module::new(self.context);
+        let module = v02::register_methods(module)?;
+        let module = v03::register_methods(module)?;
+        let module = pathfinder::register_methods(module)?;
+        let methods = module.build();
```

Comment:
Is `module` still the correct name now? (genuine question, not actually sure what else one might use)

Reply:
I updated the docstring for `Module`, apart from that I'd keep it as it wraps a struct with similar name.